bin/installtemplates/2008/Configuration_2008.ini

```diff
@@ -0,0 +1,48 @@
+[SQLSERVER2008]
+PID="$Key"
+INSTANCEID="$InstanceId"
+ACTION="Install"
+FEATURES="$ProductString"
+HELP="False"
+INDICATEPROGRESS="True"
+QUIET="True"
+QUIETSIMPLE="False"
+X86="$X86"
+PCUSOURCE="$SPSource"
+CUSOURCE="$CUSource"
+ERRORREPORTING="False"
+INSTALLSHAREDDIR="C:\Program Files\Microsoft SQL Server"

```

Comment:
Can you make these dynamic as well? So add it as a parameter and say

````powershell
[string]$InstallSharedDirectory = "C:\Program Files\Microsoft SQL Server"
````

Then 
````powershell
INSTALLSHAREDDIR=$InstallSharedDirectory
````